packages/storybook-vue/src/stories/Textbox/TextboxTypes.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Textbox } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-textbox-types" title="Vue.js Component/Textbox/Types" component={Textbox} argTypes={argTypes} />
+
+# Textbox Types
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Textbox },
+  setup() {
+    return { args };
+  },
+  template: `<Textbox v-bind="args" @change="action" />`,
+  methods: { action: action("change") },
+});
+
+## Text
+
+<Canvas>
+  <Story
+    name="Textbox Text"
+    args={{
+      placeholder: "Textbox Text",
+      name: "textbox-text",
+      id: "textbox-text",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textbox placeholder="Textbox Text" name="textbox-text" id="textbox-text" type="text"/>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Email
+
+<Canvas>
+  <Story
+    name="Textbox Email"
+    args={{
+      placeholder: "Textbox Email",
+      name: "textbox-email",
+      id: "textbox-email",
+      type: "email",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textbox placeholder="Textbox Email" name="textbox-email" id="textbox-email" type="email" />`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Password
+
+<Canvas>
+  <Story
+    name="Textbox Password"
+    args={{
+      placeholder: "Textbox Password",
+      name: "textbox-password",
+      id: "textbox-password",
+      type: "password",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textbox placeholder="Textbox Password" name="textbox-password" id="textbox-password" type="password"/>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Number
+
+<Canvas>
+  <Story
+    name="Textbox Number"
+    args={{
+      placeholder: "Textbox Number",
+      name: "textbox-number",
+      id: "textbox-number",
+      type: "number",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textbox placeholder="Textbox Number" name="textbox-number" id="textbox-number" type="number"/>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable of={Textbox} />
```

Comment:
I agree it is nice and useful to display the `type` variants here, but for completeness sake I think we should use this PR to update the textbox implementation to render all the relevant class names. You can use the React component as reference:

https://github.com/nl-design-system/utrecht/blob/main/packages/component-library-react/src/Textbox.tsx#L33-L38

Reply:
I created a [PR](https://github.com/nl-design-system/utrecht/pull/1119) to enable the related style from the css